src/python/pants/backend/docker/registries.py

```diff
@@ -24,6 +24,17 @@ def __init__(self, message):
         )
 
 
+class DockerRegistryAddressCollisionError(DockerRegistryError):
+    def __init__(self, first, second):
+        message = (
+            f"Registry addresses must be unique: {first.alias}, {second.alias}. "
+            "Ensure that each registry in `[docker]` configuration has a unique "
+            "`address` field."
+        )
```

Comment:
```suggestion
        message = softwrap(
            f"""
            Duplicated docker registry address for aliases: {first.alias}, {second.alias}.
            
            Each registry `address` in `[docker].registries` must be unique.
            """
        )
```

The term "field" is associated with values on targets defined in BUILD files only. Suggested rephrase to hopefully make it more clear as I also found the presentation of the conflicting aliases confusing.

Reply:
Could even consider including the actual address value in question in the error message to give even more context.